File: docs/src/testers/browser-controls/reload.ts
// @sniptest filename=reload.ts
// @sniptest show=1-9
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();

const session = client.Session();
await session.use(async () => {
  await session.execute({ type: "goto", url: "https://example.com" });
  await session.execute({ type: "reload" });
});

const status = await session.status();

export { status };

File: docs/src/testers/browser-controls/reload.py
# @sniptest filename=reload.py
# @sniptest show=1-7
from notte_sdk import NotteClient

client = NotteClient()

with client.Session() as session:
    session.execute(type="goto", url="https://example.com")
    session.execute(type="reload")

status = session.status()
